pandas/tseries/resample.py

```diff
@@ -567,8 +579,13 @@ def f(self, _method=method, *args, **kwargs):
 # groupby & aggregate methods
 for method in ['count', 'size']:
 
-    def f(self, _method=method):
-        return self._downsample(_method)
+    if method == "size":
```

Comment:
I don't think any of this is necessary, ``_wrap_results`` can handle all of this